### PR TITLE
Links: open everything in new tabs

### LINKED ISSUE #281: React UI: Open pulls in a new tab
Clicking on a pull card opens the pull request in the same tab. Let's keep the old behavior and have pulls open in a new tab.

### PR DESCRIPTION
**Pull request by danielbeardsley**
Users almost always want to keep pulldasher open, so let's help them
out.

Closes #281

### CODE REVIEW

File: frontend/src/pull-card/flags.tsx [hunk 1]
```diff
@@ -62,6 +62,7 @@ function PullFlag({variant, title, href, icon}: PullFlagProps) {
    return (href ?
    <Link
       sx={styles}
+      isExternal={true}
```
Comment (ardelato) [inline frontend/src/pull-card/flags.tsx:65]: Is setting `isExternal` equal to `true` for explicitness? I see from the docs we can just declare the prop -- https://chakra-ui.com/docs/components/navigation/link

External Link

**Review by ardelato — APPROVED** [review]
CR 👍

**Comment by ardelato** [comment]
Is there an issue or pull request for adding links to the `CR` and `QA` stamps? I realized we currently don't have links for them like in the old UI.


https://user-images.githubusercontent.com/22064420/162477927-228e3cf1-a779-4a8c-a076-c67b21024e84.mp4

File: frontend/src/pull-card/flags.tsx (hunk 1 above)
Comment (danielbeardsley) [reply frontend/src/pull-card/flags.tsx:65]: TIL, OMG, I didn't know this!

**Comment by danielbeardsley** [comment]
> Is there an issue or pull request for adding links to the `CR` and `QA` stamps? I realized we currently don't have links for them like in the old UI.

Make one! Let's do it!

**Comment by Michelle5102** [comment]
CR :octocat:

**Comment by ardelato** [comment]
CR 👍

**Comment by deltuh-vee** [comment]
QA 🎬 
Links for cards open in new tabs.

**Pull request merged**